src/main/cpp/utilities/bufferutility.h

```diff
@@ -20,44 +20,111 @@
  * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
-#ifndef __BUFFER_UTILITY_H_INCLUDED__
-#define __BUFFER_UTILITY_H_INCLUDED__
+#ifndef __MWENGINE__BUFFER_UTILITY_H_INCLUDED__
+#define __MWENGINE__BUFFER_UTILITY_H_INCLUDED__
 
 #include "global.h"
 #include <vector>
 
-/**
- * BufferUtility provides several functions
- * for basic time / BPM related calculations
- * where musical concepts are translated into buffers
- */
+namespace MWEngine {
 class BufferUtility
 {
     public:
 
-        /* time related math */
+        /* size calculation */
 
```

Comment:
Copied comments from Wiki page to add inline documentation on these methods' use cases.